jenkins/platform-prod/Jenkinsfile

```diff
@@ -0,0 +1,75 @@
+pipeline {
+    agent any
+    environment {
```

Comment:
Just curious about why one Jenkinsfile is inside the Netmanager folder and others are at the base of the entire monorepo....no opinion/suggestion yet, just curious.

Reply:
Good point @Baalmart , let me clean it up